js/client-chat.js

```diff
@@ -1266,13 +1266,16 @@
 					break;
 
 				case 'uhtml':
-					this.$chat.append('<div class="notice uhtml-' + toId(row[1]) + '">' + Tools.sanitizeHTML(row.slice(2).join('|')) + '</div>');
-					break;
-
 				case 'uhtmlchange':
 					var $elements = this.$chat.find('div.uhtml-' + toId(row[1]));
-					if (!$elements.length) break;
-					$elements.html(Tools.sanitizeHTML(row.slice(2).join('|')));
+					var html = row.slice(2).join('|');
+					if (!toId(html)) {
```

Comment:
Do we need to `toId` here?

Reply:
I did it incase of a blank space, though `trim()` would probably be better if anything